app/routes/questionnaire.py

```diff
@@ -361,6 +374,27 @@ def get_thank_you(schema, session_store, questionnaire_store):
     )
 
 
+@post_submission_blueprint.route("view-response/", methods=["GET"])
+@with_questionnaire_store
+@with_schema
+def get_view_submitted_response(schema, questionnaire_store):
+    try:
+        view_submitted_response = ViewSubmittedResponse(
+            schema,
+            questionnaire_store,
+            flask_babel.get_locale().language,
+        )
+
+    except (ViewSubmittedResponseNotEnabled, ViewSubmittedResponseExpired):
```

Comment:
Should expired response not go to thank you page?. That is where the content about it has expired will be, and that is what v2 does.

Reply:
Agree at this for the moment by Sarah until scenarios decided